### PR TITLE
Add admin login

### PR DESCRIPTION
**Pull request by athaichi**
This issue was completed by nostango and athaichi.

- Installed devise gem. Documentation for devise can be found here: https://github.com/heartcombo/devise 
- Installed views and controllers for User using devise. 
- On home page: 
    Implemented sign in/sign up buttons that use devise methods, which only appears when user is logged out. 
   Implemented sign out button that ends user session, which only appears when user is logged in. 
- Buttons will redirect to home page. 
- Any email can be used to create and log in to an account.

### CODE REVIEW

File: app/controllers/User/confirmations_controller.rb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
```
Comment (spinecone) [inline app/controllers/User/confirmations_controller.rb:1]: The User directory should be lowercase (user) as a rails standard practice.

File: db/schema.rb [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+# db:schema:load`. When creating a new database, `rails db:schema:load` tends to
+# be faster and is potentially less error prone than running all of your
+# migrations from scratch. Old migrations may fail to apply correctly if those
+# migrations use external dependencies or application code.
+#
+# It's strongly recommended that you check this file into your version control system.
+
+ActiveRecord::Schema.define(version: 2020_09_19_174716) do
+
+  create_table "admins", force: :cascade do |t|
```
Comment (spinecone) [inline db/schema.rb:15]: Let's make sure we get rid of this table before merging.

File: db/migrate/20200919174716_devise_create_users.rb [hunk 3]
```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+class DeviseCreateUsers < ActiveRecord::Migration[6.0]
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
```
Comment (spinecone) [inline db/migrate/20200919174716_devise_create_users.rb:6]: Is there anything else we might want on this table, like a display/ username? We'll also probably want a flag for whether or not they're an admin.

**Comment by spinecone** [comment]
Let's merge this after addressing the forgotten admins table, the User directory, and adding an admin field to the users table, and we get one reviewer's 👍

**Review by cvrkota — APPROVED** [review]

**Pull request merged**